addons/mail/static/tests/owl/test_utils.js

```diff
@@ -8,6 +8,7 @@ const EnvService = require('mail.service.Env');
 const StoreService = require('mail.service.Store');
 const Discuss = require('mail.widget.Discuss');
 const MessagingMenu = require('mail.widget.MessagingMenu');
+const concurrency = require('web.concurrency');
```

Comment:
should be grouped with `web` modules, alphabetically ordered by module names (so between `web.Class` and `web.NotificationService`)